### ISSUE TITLE
debug logs too verbose

### ISSUE THREAD

**Issue by fmassot** [quickwit-inc/quickwit#725]
There is two things that make it hard to analyze logging:

1. In Kafka source
```
debug!(
    topic = ?message.topic(),
    partition_id = ?message.partition(),
    offset = ?message.offset(),
    timestamp = ?message.timestamp(),
    num_bytes = ?message.payload_len(),
    doc = ?doc.as_str(),
    "Message received.",
);
```
We can maybe log that but without the message, or just take the first 30 characters.

2. And the fact that `RawDocBatch` is logged too. We need to set how `RawDocBatch` is displayed, by default it is showing all docs...

**Comment by guilload** [comment]
Shortening the message for 1. makes sense. With the partition ID and offset, you can always get the message directly in Kafka if you really need to.

**Comment by fmassot** [comment]
Fixed in #732
